### PR TITLE
[HTD-36] Feat: Add `Integration cta / hero` component

### PR DESCRIPTION
**Pull request by RBoraurito**
## Assets

| Light | Dark |
| ------ | ----- |
|    image    |   image    |

## Before Merge Checklist

The following steps were checked before merging a PR

- [x] checked and fixed code style issues
- [x] checked and removed unnecessary/junky code
- [x] manually tested introduced solution
- [ ] added needed tests to automate testing this code in the future

### CODE REVIEW

File: src/components/sections/integration/cta/components/icon-rows/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const parserOffset = (offset: number) => {
+  const additionalOffset = Math.floor(offset) * 0.16; // .16 is the sixth part of the icon size (72)
+  return offset + additionalOffset;
+};
+
+export const IconRows = ({ rows, $orientation, $offset }: IconRowsProps) => {
+  return (
+    <StyledIconRowsWrapper $offset={$offset} $orientation={$orientation}>
+      {Array.isArray(rows) &&
+        rows?.length > 0 &&
```
Comment (nephlin7) [inline src/components/sections/integration/cta/components/icon-rows/index.tsx:15]:
```suggestion
      {Array.isArray(rows)?.length > 0 &&
```

**Review by nephlin7 — APPROVED** [review]

**Review by nephlin7 — APPROVED** [review]
Great job @RBoraurito 👏🏽 !

**Comment by RBoraurito** [comment]
@nephlin7 This section is on draft since I don't have the right icons, should I wait, or can I send it?

**Comment by nephlin7** [comment]
> @nephlin7 This section is on draft since I don't have the right icons, should I wait, or can I send it?

Yeah, let's wait for them, please. Move the ticket to blocked and leave a comment on why is on that status.

**Review by nephlin7 — APPROVED** [review]

**Pull request merged**